src/js/factory/calendar.js

```diff
@@ -434,8 +442,11 @@ var mmin = Math.min;
  * @property {boolean} [useCreationPopup=false] - Whether use default creation popup or not. The default value is false.
  * @property {boolean} [useDetailPopup=false] - Whether use default detail popup or not. The default value is false.
  * @property {Array.<Timezone>} [timezones] - {@link Timezone} array.
- *  The first Timezone element is primary and can override Calendar#setTimezoneOffset function
+ *  The first Timezone element is primary
  *  The rest timezone elements are shown in left timegrid of weekly/daily view
+ * @property {function} [timezoneOffsetFn] - If the browser does not support Intl.DateTimeFormat and formatToParts APIs(such as IE11 and below), it is recommended to register function('timezoneOffsetFn' property) to calculate and pass the offset outside the application.
```

Comment:
IE11 => Internet Explorer 11

Reply:
"it is recommended to register function('timezoneOffsetFn' property) to calculate and pass the offset outside the application." => Can be short. "it is recommended to provide timezone offset via 'timezoneOffsetFn'"